chromadb/__init__.py

```diff
@@ -181,6 +181,53 @@ def HttpClient(
     return ClientCreator(tenant=tenant, database=database, settings=settings)
 
 
+def CloudClient(
+    tenant: str,
+    database: str,
+    api_key: Optional[str] = None,
+    settings: Optional[Settings] = None,
+) -> ClientAPI:
+    """
+    Creates a client to connect to a tennant and database on the Chroma cloud.
+
+    Args:
+        tenant: The tenant to use for this client.
+        database: The database to use for this client.
+        api_key: The api key to use for this client.
+    """
+
+    CLOUD_HOST = "api.trychroma.com"
+    CLOUD_PORT = "443"
+
+    # If no API key is provided, try to load it from the environment variable
+    if api_key is None:
+        import os
+
+        api_key = os.environ.get("CHROMA_API_KEY")
+
+    # If the API key is still not provided, prompt the user
+    if api_key is None:
+        print(
+            "\033[93mDon't have an API key?\033[0m Get one at https://app.trychroma.com"
+        )
+        api_key = input("Please enter your Chroma API key: ")
```

Comment:
I have mixed feelings about this. On one hand it's a great user experience for devs working locally. On the other hand it will completely block an actual production binary. WDYT @HammadB ?

Reply:
I think if you're deploying this in prod, and you don't have an API key set, you can look in your logs to see what's going on, and you will see this output. I'll add a logger warning to make sure there's also a log statement.